container-images/scripts/build_llama_and_whisper.sh

```diff
@@ -29,6 +58,9 @@ main() {
   mv build/bin/server "$install_prefix/bin/whisper-server"
   cd ..
   rm -rf whisper.cpp
+
+  CMAKE_ARGS="${common_flags[*]}" FORCE_CMAKE=1 \
+    pip install --prefix="$install_prefix" 'llama-cpp-python[server]'
```

Comment:
This is the significant change in this PR, the rest is just address maintenance concerns from a previous PR and doing further de-duplication.